gateway/tests/api/test_files.py

```diff
@@ -30,25 +31,17 @@ def test_files_list(self):
         media_root = os.path.normpath(os.path.join(os.getcwd(), media_root))
 
         with self.settings(MEDIA_ROOT=media_root):
-            auth = reverse("rest_login")
-            response = self.client.post(
-                auth, {"username": "test_user", "password": "123"}, format="json"
-            )
-            token = response.data.get("access")
-            self.client.credentials(HTTP_AUTHORIZATION="Bearer " + token)
+            user = models.User.objects.get(username="test_user")
```

Comment:
I would like to understand more in-depth this change because until now we had two authentication mechanisms: `mock_token` & `custom_token` (ref [here](https://github.com/Qiskit-Extensions/quantum-serverless/blob/28cf8112133ed2d66ea4e88829dace0a63d42996/gateway/README.md?plain=1#L25)).

with this change are we going to loose this functionality? (Sorry, I couldn't test it yet I can take a look more in-depth next week)

Reply:
They should not be affected these changes.  This test is using `/dj-rest-auth/login` endpoint that is taken out in this PR.  So I had to use another way to access the endpoint that this test is testing.  Would you check if the `custom_token` authentication is OK?

Reply:
Sure, I will take a look monday on this and let you know @akihikokuroda , thank you!

Reply:
Thanks!

Reply:
Everything working in my side 🎉